specs/sql/migrate-from-mysql-database-to-postgressql-database.yaml

```diff
@@ -0,0 +1,42 @@
+---
+name: Migrate from MySQL Database to PostgresSQL Database
+command: "pgloader mysql://{{mysql_username}}:{{mysql_password}}@{{mysql_hostname}}:{{mysql_port}}/{{mysql_database_name}} postgresql://{{postgres_username}}:{{postgres_password}}@{{postgres_hostname}}:{{postgres_port}}/{{postgres_database_name}}"
+tags:
+  - postgresql
+  - mysql
+description: Loads data from the MySQL database, transforms it to postgresql compatible format and uploads it to the PostgresSQL database. Note -  it requires pgloader to be installed and both PostgresSQL and MySQL hosts are accessible from the machine running this script.
```

Comment:
uber nit: could we be consistent between `PostgreSQL` and `postgresql`?

Reply:
Thanks for catching that! I have updated it to `PostgresSQL`